packages/lesswrong/components/walledGarden/GardenCodesItem.tsx

```diff
@@ -20,47 +40,90 @@ const styles = theme => ({
   eventTime: {
     ...eventTime(theme)
   },
-  icon: {
+  personalIcon: {
+    ...iconStyling,
+    marginRight: "3px",
```

Comment:
Style nitpick: this works fine, but in JSS we usually write this as numbers without the units, ie `3` instead of `"3px"`. (In CSS, the units are required; in JSS, the units are px unless otherwise specified.)

Reply:
Ah, noted. I'll fix in my next PR. Cheers.